### ISSUE TITLE
Training loss chart

### ISSUE THREAD

**Issue by chtham** [Megvii-BaseDetection/YOLOX#1356]
How I can plot a graph of those Training lossing chart

**Comment by FateScript** [comment]
You might get such a plot via tensorboard. It's supported by default.

**Comment by chtham** [comment]
> You might get such a plot via tensorboard. It's supported by default.

Where I can get the coding to plot? I have one tensorboard/events.out.tfevents.1654151500.d9337b6d73cd.674.0 in the output after I run the coding of training. But I dk how to load it .

**Comment by FateScript** [comment]
@chtham  run `tensorboard --logdir /path/to/your/log_dir` in your cli. Plz read the doc: https://www.tensorflow.org/tensorboard/get_started for more details.
